src/win/util.c

```diff
@@ -175,9 +172,6 @@ int uv_cwd(char* buffer, size_t* size) {
     return uv_translate_sys_error(GetLastError());
   }
 
-  /* utf16_len contains the length, *not* including the terminating null. */
-  utf16_buffer[utf16_len] = L'\0';
-
   /* The returned directory should not have a trailing slash, unless it points
    * at a drive root, like c:\. Remove it if needed. */
   if (utf16_buffer[utf16_len - 1] == L'\\' &&
```

Comment:
This is an OOB read

Reply:
Oh, that's indeed a sneaky bug, good catch. This PR doesn't make it worse so I'll go ahead and merge it, then follow up with another one.